### DISCUSSION TITLE
custom template

### DISCUSSION THREAD

**Discussion by gogi100** [rejetto/hfs#128] [Q&A]
can i connect new version the hfs v0.36 with template for  v2.3i build 297 of the hfs? if this possible does not exists, how customize customizing template?

**Comment by rejetto** [comment]
HFS 2 templates are not compatible with HFS 3.
In HFS 3 there's no template like before. You can only ADD things.
see https://github.com/rejetto/hfs/discussions/127#discussioncomment-5188127

i will add more possibilities as i understand it's necessary.

Someone was making a plugin to load hfs2 templates in hfs3, but it's not part of this project.
